### PR TITLE
Add Next.js

### PR DESCRIPTION
**Pull request by ooade**
**https://github.com/unicodeveloper/awesome-nextjs**

**Curated list of resources: books, videos, articles about using Next.js.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

- I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- This pull request has a descriptive title.<br>For example, `Add Name of List`, not `Update readme.md` or `Add awesome list`.
- The entry in the Awesome list should:
	- Include a short description about the list project/theme. It should not describe the list itself.<br>Example: `- [Fish](…) - User-friendly shell.`
	- Be added at the bottom of the appropriate category.
- The list I'm submitting complies with these requirements:
	- **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
	- It's the result of hard work and the best I could possibly produce.
	- Non-generated Markdown file in a GitHub repo.
	- The repo should have the following [GitHub topics](https://help.github.com/articles/about-topics) set: `awesome-list`, `awesome`, `list`. I encourage you to add more relevant topics.
	- Not a duplicate.
	- Includes a succinct description of the project/theme at the top of the readme. [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- Only has awesome items. Awesome lists are curations of the best, not everything.
	- Includes a project logo/illustration whenever possible.
		- Placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
		- The image should link to the project website or any relevant website.
		- The image should be high-DPI. Set it to maximum half the width of the original image.
	- Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
	- Has the [Awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- Has a Table of Contents section.
		- Should be named `Contents`, not `Table of Contents`.
		- Should be the first section in the list.
	- Has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license).
		- That means something like CC0, **not a code licence like MIT, BSD, Apache, etc.**
		- If you use a license badge, it should be SVG, not PNG.
	- Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines).
		- The file should be named `contributing.md`. Casing is up to you.
	- Has consistent formatting and proper spelling/grammar.
		- Each link description starts with an uppercase character and ends with a period.<br>Example: `- [AVA](…) - JavaScript test runner.`
		- Drop all the `A` / `An` prefixes in the descriptions.
		- Consistent naming. For example, `Node.js`, not `NodeJS` or `node.js`.
	- Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- Go to the top and read it again.

### CODE REVIEW

**Comment by legodude17** [comment]
Needs descriptions for all of the items.

**Comment by ooade** [comment]
@legodude17 Description for artcles??? Don't think so. It's on our contributing.md

**Comment by legodude17** [comment]
From the requirements:
> Entries have a description, unless the title is descriptive enough by itself. It rarely is though.

For example, describe the articles in the articles section and say what they talk about.

**Comment by ooade** [comment]
Our [contributing.md](https://github.com/unicodeveloper/awesome-nextjs/blob/master/CONTRIBUTING.md)
> Articles should use the [Title Case Name](link) format.

**Comment by ooade** [comment]
@sindresorhus ???

File: readme.md [hunk 1]
```diff
@@ -209,6 +209,7 @@
 - [WebGL](https://github.com/sjfricke/awesome-webgl) - JavaScript API for rendering 3D graphics.
 - [Preact](https://github.com/ooade/awesome-preact) - App framework.
 - [Progressive Enhancement](https://github.com/jbmoelker/progressive-enhancement-resources)
+- [Next.js](https://https://github.com/unicodeveloper/awesome-nextjs) - Framework for server-rendered React apps.
```
Comment (sindresorhus) [inline readme.md]: Invalid URL

**Comment by sindresorhus** [comment]
- `Next.js` in `Next.js is a minimalistic framework for server-rendered React applications.` should be linkified to the Next.js repo.
- `Github` => `GitHub`
- Remove the emojis in the descriptions. They're just distracting.
- `next.js` => `Next.js`
- `Next on NextJS` => `Next on Next.js`
- Missing the `awesome` GitHub topic.
- `HackerNews` => `Hacker News`
-  `TodoList` => `Todo list`
- `Simple markdown` => `Simple Markdown`
- This descriptions needs more work `⚡️ RAN! React . Apollo (GraphQL) . Next.js Boilerplate ⚡️ - SEO-Ready, Production-Ready, SSR, Hot-Reload, CSS-in-JS, Caching and more with CLI easy-to-use` It's also missing a dot at the end.

**Comment by sindresorhus** [comment]
> Drop all the `A` / `An` prefixes in the descriptions.

⬆️

**Comment by ooade** [comment]
Will Re-open or fix those :sweat_smile:

**Comment by Sly777** [comment]
@sindresorhus is this okay?

> * [RAN!](https://github.com/sly777/ran) - SEO-Ready, Production-Ready boilerplate with Apollo (GraphQL) support. It's ready to use SSR, Hot-Reload, CSS-in-JS, Caching and more with CLI for easy-to-use.

**Comment by sindresorhus** [comment]
Could still be improved:

> [RAN!](https://github.com/sly777/ran) - Production-ready boilerplate with support for GraphQL, SSR, Hot-reload, CSS-in-JS, caching, and more.

**Comment by Sly777** [comment]
I updated my PR with this ;)

**Pull request closed without merging**